tests/test-cases/Group3-Docker-Compose/3-03-Docker-Compose-Basic.robot

```diff
@@ -32,73 +33,21 @@ Check Compose Logs
-    Log  ${output}
-    Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file basic-compose.yml start
-    Log  ${output}
-    Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file basic-compose.yml logs
-    Log  ${output}
-    Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file basic-compose.yml stop
-    Log  ${output}
-    Should Be Equal As Integers  ${rc}  0
-
-Compose kill
-    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f basic-compose.yml up -d
-    Log  ${out}
-    Should Be Equal As Integers  ${rc}  0
-
-    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f basic-compose.yml kill redis
-    Log  ${out}
-    Should Be Equal As Integers  ${rc}  0
-
-    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f basic-compose.yml down
-    Log  ${out}
-    Should Be Equal As Integers  ${rc}  0
-
-Compose Up while another container is running (ps filtering related)
-    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d busybox /bin/top
-    Log  ${out}
-    Should Be Equal As Integers  ${rc}  0
-
-    ${rc}  ${out}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f basic-compose.yml up -d
-    Log  ${out}
-    Should Be Equal As Integers  ${rc}  0
-
-Compose Up with link
-    Run  echo '${link-yml}' > link-compose.yml
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file link-compose.yml up -d
-    Log  ${output}
+Compose up in foreground (attach path)   
+    Run  echo '${hello-yml}' > hello-compose.yml
+    ${rc}  ${output}=  Run And Return Rc And Output  mkfifo /tmp/fifo
     Should Be Equal As Integers  ${rc}  0
-    Wait Until Keyword Succeeds  10x  10s  Check Compose Logs
-
-Compose bundle creation
-    ${rc}  Run And Return Rc  docker-compose %{COMPOSE-PARAMS} --file basic-compose.yml pull
+    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} -f hello-compose.yml pull
     Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file basic-compose.yml bundle
     Log  ${output}
-    Should Contain  ${output}  Wrote bundle
-    Should Be Equal As Integers  ${rc}  0
 
-Compose up -d --force-recreate
-    Run  echo '${rename-yml-1}' > compose-rename.yml
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file compose-rename.yml up -d
-    Log  ${output}
-    Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${output}=  Run And Return Rc And Output  docker-compose %{COMPOSE-PARAMS} --file compose-rename.yml up -d --force-recreate
```

Comment:
not really understanding here why we are removing the new force-restart that we just implemented?